tests_peerscout_api/unit_tests/main_test.py

```diff
@@ -63,6 +65,8 @@
     }
 }
 
+EDITOR_TYPE_MOCK = '{editor_type}'
```

Comment:
Is this meant to be just an example, maybe something without curly brackets in case they affect the outcome? e.g. `EditorType1` or something like that?